app.js

```diff
@@ -4,5 +4,23 @@ const app = express()
 const controller = require("./controller")
 
 app.get("/api/topics", controller.getTopics)
+app.get("/api/articles/:article_id", controller.getArticleById)
+app.get("/api/users", controller.getUsers)
```

Comment:
Now you have endpoints with /topics , /articles and /users, you could look at separating them out into separate models controllers (not mandatory though!)